feat(task): fluent expression WIP (#26)
Guikingone/SchedulerBundle: doc/best_practices.md

# Best practices

- [Transports](#Transports)

> - [External ressources](#Others)

Reviewer: ```suggestion
- [External ressources](#external-resources)
```

Author: Fixed

## Transports

> When using transports that rely on network calls, you must keep in mind that this transport

Reviewer: ```suggestion
When using transports that rely on network calls, you must keep in mind that these transports
```

Author: Fixed

can fail to retrieve/create and so on, it could be a good idea to use the fail over or round robin
transport to prevent any errors:

```yaml
scheduler_bundle:
    transport:
        dsn: 'failover://(doctrine://... || fs://last_in_first_out)'
```

> ## Others

Reviewer: ```suggestion
## External resources
```

Author: Fixed
